src/test.zig

```diff
@@ -5,5 +5,8 @@ test {
     testing.refAllDeclsRecursive(@import("main.zig"));
     testing.refAllDeclsRecursive(@import("snapshot.zig"));
     testing.refAllDeclsRecursive(@import("builtins/main.zig"));
+
+    // TODO: Remove after hooking up
```

Comment:
Should this be removed now? 😄

Reply:
Since this module isn't yet used in any codepath, it'll be DCE'd. But i'll remove in a subsequent PR that hooks `occurs` up into `unify`!